hack/ginkgo-e2e.sh

```diff
@@ -56,6 +59,7 @@ else
     prepare-e2e
 
     detect-master >/dev/null
+    KUBE_MASTER_URL="${KUBE_MASTER_URL:-https://${KUBE_MASTER_IP:-}}"
```

Comment:
What is consuming this variable?
I would have expected the `--host` line below to use it, and for it to be called KUBE_MASTER_HOST to match the parameter...